### PR TITLE
Internationalize content

### PR DESCRIPTION
**Pull request by rafaelrpbelo**
- Internationalize whole content
- Organize locale files' struct

### CODE REVIEW

**Comment by rafaelrpbelo** [comment]
@dmitryrck Can you review? 😉

**Comment by dmitryrck** [comment]
Yep :D

File: app/views/customers/index.html.erb [hunk 1]
```diff
@@ -1,18 +1,18 @@
-<h1>Listagem de clientes</h1>
+<h1><%= t('.title') %></h1>
 
 <table>
   <thread>
     <tr>
-      <th>Nome</th>
-      <th>CPF</th>
+      <th><%= t('.list.name') %></th>
+      <th><%= t('.list.cpf') %></th>
```
Comment (dmitryrck) [inline app/views/customers/index.html.erb:7]: Why don't you use `Customer.human_attribute_name :subject`? And in the locale:

```
en:
  activerecord:
    attributes:
      customer:
        name: "Name"
```

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
-    <div class="navbar-header">
-      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false">
-        <span class="sr-only">Toggle navigation</span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-        <span class="icon-bar"></span>
-      </button>
-      <%= link_to 'CarDealership', root_path, class: 'navbar-brand' %>
-    </div>
+<% I18n.with_options scope: 'navbar' do |i18n| %>
```
Comment (dmitryrck) [inline app/views/shared/_navbar.html.erb:1]: This is new for me :ok_hand:

File: app/views/customers/index.html.erb (hunk 1 above)
Comment (rafaelrpbelo) [inline app/views/customers/index.html.erb:7]: This is a solution, but I think that field doesn't belongs to the attribute name, but something like the table field. Like the example https://github.com/rafaelrpbelo/car_dealership_sm/blob/internationalize-content/app/views/vehicles/index.html.erb#L8, the year_model table field isn't related to any vehicle field.

I think this way I can keep a standard easily. But your suggestion is too welcome.

Comment (dmitryrck) [inline app/views/customers/index.html.erb:7]: Well… I don't think so… you are using an attribute from vehicle…

Answer this question:

What happen if you have two tables… one for vehicle and another for order, in both you will display the color of the vehicle, but in vehicle table you name it _color_ and in order table you name it _vehicle's color_…

?

Comment (rafaelrpbelo) [inline app/views/customers/index.html.erb:7]: Yes, but what do you suggest when I need to leave out from the human attribute's pattern? Like [this](https://github.com/rafaelrpbelo/car_dealership_sm/blob/internationalize-content/app/views/vehicles/index.html.erb#L8).
Using attribute's name I just can think about:

``` html
...
<th><%= Vehicle.human_attribute(:brand) %></th>
<th><%= Vehicle.human_attribute(:version) %></th>
<th><%= t('.list.year_model') %></th>
...
```

Comment (dmitryrck) [inline app/views/customers/index.html.erb:7]: :okay: it is up to you :P 

your project, your rules :P

**Pull request merged**